site/shared/src/main/scala/shared/User.scala

```diff
@@ -26,9 +26,8 @@ case class UserProgress(
 
 case class OverallUserProgressItem(
   libraryName: String,
-  sections:    Int,
-  completed:   Int,
-  color:       String
+  sections:    Long,
+  completed:   Boolean
```

Comment:
Note that `completed` cannot be inferred from the progress alone, we need to know if we have completed as many sections as they are available and the list of sections is only known at runtime.